src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -85,7 +86,7 @@ abstract class AbstractCheckTask(
 
     protected var handledMessageCounter: Long = 0
 
-    protected val converter = ProtoToIMessageConverter(VerificationUtil.FACTORY_PROXY, null, null)
+    protected val converter = ProtoToIMessageConverter(VerificationUtil.FACTORY_PROXY, null, null, createParameters().setUseMarkerForNullsInMessage(true))
```

Comment:
Can we make the `converter` as a static field?

Reply:
Yep, I think there is no problem to do that.